core/scripts/local/DeployEMP.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
nit: I think this should also be a separate PR.

Reply:
deleted